### DISCUSSION TITLE
How to export the pola figure data by xrd?

### DISCUSSION THREAD

**Discussion by Xinqi777** [mtex-toolbox/mtex#2192] [Ask Anything]
Hello everyone, I am a programming white. I have successfully run the scatterlike pole plot with mtex. But the contour lines and odf calculations don't match very well. I want to export the data into  the origin class to obtain the contour lines data . What code should I use? Thanks.

![1723600223951](https://github.com/user-attachments/assets/f1e8bf14-b5cc-45af-9e3d-d64561e26185)

**Comment by ralfHielscher** [comment]
My advice: do not waste your time making your data looking prettier. Try to answer the questions you are really interested in.
Ralf.
